integreat_compass/cms/views/offers/offer_form_view.py

```diff
@@ -32,7 +32,7 @@ class OfferFormView(TemplateView):
     def _get_forms(offer_id, **kwargs):
         if offer_id:
             offer = Offer.objects.get(id=offer_id)
-            offer_version = offer.versions.first()
+            offer_version = list(offer.versions.select_related())[-1]
```

Comment:
This is incredibly ugly, but I could not figure out a different way. The ordering won't budge when using `order_by`, and using `reverse` on the selected offers has no effect either.

Any ideas how to improve this?

Reply:
Update:  #71 uses the same approach but inside a latest_version property, so probably move the discussion to there.